docs/Glossary.md

```diff
@@ -69,7 +69,7 @@ A *hash* is a string that represents the hash portion of the URL. It is synonymo
 type LeaveHook = (prevState: RouterState) => any;
 ```
 
-A *leave hook* is a user-defined function that is called when a route is about to be unmounted. It receives the previous [router state](#routerstate) as its first argument. 
+A *leave hook* is a user-defined function that is called when a route is about to be unmounted. It receives the previous [router state](#routerstate) as its first argument.
 
```

Comment:
oh, there is a diff here, it was just the 'trailing spaces'. If there is a problem, I can revert.